projects/openvswitch/project.yaml

```diff
@@ -0,0 +1,7 @@
+homepage: "http://www.openvswitch.org"
+primary_contact: "blp@ovn.org"
+auto_ccs:
+  - "jpettit@ovn.org"
```

Comment:
It may be helpful for your email to be here so that if there is a build failure you will be notified.
I guess it is up to the project maintainers if they want you to have access to security bugs as well.

Reply:
I am testing locally if everything works. Hopefully, adding me in CC won't be necessary.

Reply:
OK